### PR TITLE
created a method to get logs

### PR DESCRIPTION
**Pull request by ajaykumarraghav**
@ankitjain0105 @pratyush23 @anthonyuma @DitmirH @dapoawoola added a method to get the errors and warnings from console.Please review.

### CODE REVIEW

File: lib/frameworks/utils.rb [hunk 1]
```diff
@@ -95,5 +95,34 @@ def save_and_link_screenshot
       Capybara.current_session.driver.save_screenshot("./reports/Screenshot_#{current_time}.png")
       embed "./reports/Screenshot_#{current_time}.png", 'image/png', "Actual screenshot of the error at #{current_url}"
     end
+
+    def getting_console_errors(type)
```
Comment (pratyush23) [inline lib/frameworks/utils.rb]: Please change the method name to get_console_logs

File: lib/frameworks/utils.rb [hunk 2]
```diff
@@ -95,5 +95,34 @@ def save_and_link_screenshot
...
+       end
+     when 'warnings'
+       warnings = page.driver.browser.manage.logs.get(:browser).select {|e| e.level != "SEVERE"}.map(&:message).to_a
+       if warnings.present?
+         puts "Below are the warnings found."
+         raise StandardError, warnings.join("\n\n")
+       else
+         puts "No warnings found in console"
+       end
+     when 'all'
```
Comment (pratyush23) [inline lib/frameworks/utils.rb]: 'all' is not appropriate here for 'type'. Lets discuss on this

**Review by pratyush23 — CHANGES_REQUESTED** [review]

File: lib/frameworks/utils.rb (hunk 2 above)
Comment (ankitjain0105) [reply lib/frameworks/utils.rb]: agree, we can say something 'errors_and_warnings'

File: lib/frameworks/utils.rb (hunk 1 above)
Comment (ankitjain0105) [reply lib/frameworks/utils.rb]: agree.

**Review by ankitjain0105 — CHANGES_REQUESTED** [review]

**Review by pratyush23 — APPROVED** [review]
♐️

**Review by ankitjain0105 — APPROVED** [review]

**Pull request merged**